fairscale/nn/data_parallel/fully_sharded_data_parallel.py

```diff
@@ -189,6 +190,14 @@ class FullyShardedDataParallel(nn.Module):
             module to be wrapped with FSDP.
         process_group (Optional):
             process group for sharding
+        process_group_reduce_scatter (Optional):
+            process group for reduce scatter
```

Comment:
add default value and what does it mean when it is left at the default value.

Reply:
done. Default to None.